recipes/bgen/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "bgen" %}
+{% set version = "0.1.13" %}
+{% set sha256 = "dda041bf90fc447024ddb9afb9f6932e7c083067336eebec68d24c4077f5c98c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/limix/bgen/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win32 or linux32 or (win64 and py27)]
+
+requirements:
+  build:
+    - python  # [win]
+    - conda-build
```

Comment:
This is odd. Do you need to specify `conda-build` here? It is granted that `conda-build` will be available anyways.